assets/css/style.css

```diff
@@ -1,11 +1,18 @@
 .pulse {
+	all: unset;
 	border-radius: 50%;
+	border: 0;
+	display: block;
 	position: absolute;
 	cursor: pointer;
 	width: 12px;
 	height: 12px;
 	margin: -6px;
 }
+.pulse:focus {
```

Comment:
This focus state is going to be just barely visible. It's important to remember that low-vision keyboard navigators do not know where focus is going to go while they navigate: focus changes are undirected changes of attention, where hover states are directed changes of attention. Because of that, it's very important that the visible focus is bold enough to stand out visually. Otherwise, it's very easy for a user to lose track of where they are on the page.